doc/changes/changes_3.4.0.md

```diff
@@ -1,15 +1,25 @@
-# Exasol Test Containers 3.4.0, released 2020-11-27
+# Exasol Test Containers 3.4.0, released 2020-12-01
 
 Code name: Matrix testing
 
 In the 3.4.0 release we added the ability to override the docker image used via the Java system property `com.exasol.dockerdb.image`. You can use this in a continuous integration build for example to iterate through a list of Exasol versions that you want to test your software against.
+
+The release also now uses `project-keeper` to maintain uniform project setup and fixes some bugs and code smells.
  
 ## Features
  
-* #105: Fixed code smells.
 * #106: Docker image can be overridden by property.
+
+## Bugfixes
+
+* #105: Fixed code smells.
 * #109: Adding "JDBC" as service in `withRequiredServices()` does not throw exception anymore.
 
+## Refactoring
+
+* #106: Introduced CI matrix test with Exasol 6.1.12 and 7.0.4
+* #112: Using `project-keeper` to manage common parts.
+
```

Comment:
the new dependency is missing

* Added `com.exaso.l:project-keeper-maven-plugin:0.4.1`

Reply:
Also, the new dependency is missing in the README file.

`| [Project Keeper Maven Plugin](https://github.com/exasol/project-keeper-maven-plugin)| Checking project structure                             | MIT License                   |`

Reply:
You are absolutely right.